src/Optimization/abstract_model_store.jl

```diff
@@ -30,10 +30,6 @@ abstract type AbstractModelStore end
 # - :expressions
 
 # Required methods for subtypes:
-# - initialize_storage!
```

Comment:
There is something I found confusing but did not try to fix. PSI imports IS.Optimization.read_optimizer_stats but not these others. So, implements its own version of that method but then creates PSI methods for the others. There is not a clean separation of what is PSI and what is IS.Optimization.